docs/_source/docs/templates.rst

```diff
@@ -190,53 +190,21 @@ to generate CloudFormation Template as a `json` string.
 AWS CDK
 ^^^^^^^
 
-This example generates a cloudformation template from `AWS CDK`_ code.
+AWS CDK can be used to programmatically generate templates for your stacks and then Sceptre can
```

Comment:
hmm, looks like we have two locations where we are referencing template handlers.  what do you think about consolidating that info.  Maybe move [list of available template handlers](https://docs.sceptre-project.org/3.2.0/docs/template_handlers.html#available-template-handlers) to this page?

Reply:
I think doing that might need a larger-scale rewrite of those two pages. I don't want to do that in this PR, though.

Reply:
sounds good